ansible/group_vars/clickhouse/vars.yml

```diff
@@ -15,6 +15,8 @@ nftables_clickhouse_allow:
     ip: 5.9.112.244
   - fqdn: clickhouseproxy.dev.ooni.io
     ip: "{{ lookup('dig', 'clickhouseproxy.dev.ooni.io/A') }}"
+  - fqdn: clickhouseproxy.prod.ooni.io
+    ip: "{{ lookup('dig', 'clickhouseproxy.prod.ooni.io/A') }}"
```

Comment:
Same as above, even moreso important here, since ideally we would not allow access to the production clickhouse from the dev environment